### PR TITLE
Updates to it

### PR DESCRIPTION
**Pull request by francematt**
some updated translations

### CODE REVIEW

**Comment by liuderchi** [comment]
this PR is for issue #93

File: package.json [hunk 1]
```diff
@@ -73,7 +77,7 @@
         },
         {
           "value": "nl",
-          "description": "Nederlands (nl)"
+          "description": "Nederlandhttps://help.github.com/articles/syncing-a-fork/https://help.github.com/articles/syncing-a-fork/https://help.github.com/articles/syncing-a-fork/https://help.github.com/articles/syncing-a-fork/s (nl)"
```
Comment (liuderchi) [inline package.json]: Is it accidentally input?

File: package.json [hunk 2]
```diff
@@ -59,6 +59,10 @@
           "value": "hi",
           "description": "हिन्दी (hi)"
         },
+                {
+          "value": "it",
+          "description": "Italiano (it)"
+        },
```
Comment (liuderchi) [inline package.json]: We've already got option `it` here (line no.67).
Please remove these changes.

**Comment by liuderchi** [comment]
@francematt Thanks so much for those hard work. 👍 

I know it really takes much time to get all done.
Please feel free to notify me when you want to merge it.

File: package.json (hunk 1 above)
Comment (francematt) [inline package.json]: yes, it is accidentally input, sorry... something got wrong...

Comment (liuderchi) [inline package.json]: some reminder: 

after you make changes, please push those new commits to 
`francematt:master` and this page will automatically loa latest code.

**Comment by liuderchi** [comment]
@francematt I will merge it forcibly then fix those typo by myself.

**Pull request merged**